app/dl_formats.py

```diff
@@ -23,14 +23,11 @@ def get_format(format: str, quality: str) -> str:
         # Quality is irrelevant in this case since we skip the download
         return "bestaudio/best"
 
-    if format == "mp3":
+    if format in ("m4a", "mp3"):
         # Audio quality needs to be set post-download, set in opts
         return "bestaudio/best"
 
-    if format in ("mp4", "any"):
```

Comment:
it seems like the "any" case is not handled anymore in this function, and would result in an exception